tier-1/rpm-packaging.yaml

```diff
@@ -4,3 +4,13 @@ packages:
   - dnf
   # To ensure we can enable client certs to access RHEL content
   - subscription-manager
+
+postprocess:
+  # These enable librhsm which enables host subscriptions to work in containers
+  # https://github.com/rpm-software-management/librhsm/blob/fcd972cbe7c8a3907ba9f091cd082b1090231492/rhsm/rhsm-context.c#L30
+  # https://github.com/openshift/os/pull/876/commits/dd35dd0e102aeed90df14f05c8ae9da4c8c5962a
+  - |
+    #!/usr/bin/bash
```

Comment:
My only comment here is to not use #!/usr/bin/bash when nothing is bash-specific.  bash is not as portable as many think and bash as shipped by Apple still is at 3.2.57 due to GPLv3 (not really hugely relevant, it's just that a large installation of bash out in the wild is 3.2.57).  At any rate, this is just my own personal nitpick.  Just say #!/bin/sh or #!/usr/bin/sh and stick with portable shell syntax when possible.  Yes, we still have to deal with that in 2024.

Reply:
True...though in practice we know we're executing the code in this container image which has the bash we want.

Ultimately this specific bit and others will get replaced with a plain `RUN` invocation in a container build which has that same semantic.